oeps/best-practices/oep-0066-bp-openfeature-toggles.rst

```diff
@@ -0,0 +1,130 @@
+.. _pep_based_template:
+
+.. Below is the display in the left sidebar on RTD. Please omit leading 0's
+
+OEP-0066: OpenFeature Format For Feature Toggles
+######################
```

Comment:
To fix the build, you'll need to make your underlines match the length of the above text.